src/french.js

```diff
@@ -409,32 +402,30 @@ class French {
       }
     };
 
-    const subject = this.getSubject (info);
+    const subject = this.getSubject(info);
 
     if (word === 'être' || word === 'avoir' || word === 'aller') {
       return specialVerbs[word][subject];
     }
 
-    const verbProperties = this.getWordGroup (word);
+    const verbProperties = this.getWordGroup(word);
     const group = verbProperties[0];
     let stem = verbProperties[1];
 
     if (group === 'group1') {
-      let solutions = this.combineToConjugate(group, stem, tenseEnding['ending1'][subject]);
+      const solutions = this.combineToConjugate(group, stem, tenseEnding.ending1[subject]);
       if (solutions.length > 1) {
         return `${solutions[0]} / ${solutions[1]}`;
-      } else {
-        return solutions[0];
       }
+      return solutions[0];
     } else if (group === 'group2') {
       if (/ï$/.test(stem) && /^s/.test(subject)) {
         stem = `${stem.slice(0, -1)}i`;
       }
-      let solutions = this.combineToConjugate(group, stem, tenseEnding['ending2'][subject]);
+      const solutions = this.combineToConjugate(group, stem, tenseEnding.ending2[subject]);
       return solutions[0];
-    } else {
-      return 'not yet implemented'
     }
+    return 'not yet implemented';
   }
 }
-export { French };
+export default { French };
```

Comment:
Wow, nice work @minsooshin.

Using default won't work here. French is a class, not a constructor.